File: C++/Classes/ClassesAndObjects.cpp
#include <iostream>

// Definindo uma classe Pessoa
class Pessoa {
public:
    std::string nome;
    int idade;

    // Método da classe
    void apresentar() {
        std::cout << "Olá, meu nome é " << nome << " e tenho " << idade << " anos." << std::endl;
    }
};

int main() {
    Pessoa pessoa1; // Criando um objeto da classe Pessoa
    pessoa1.nome = "Artur";
    pessoa1.idade = 30;
    pessoa1.apresentar();

    Pessoa pessoa2;
    pessoa2.nome = "Maria";
    pessoa2.idade = 25;
    pessoa2.apresentar();

    return 0;
}

File: PHP/POO/Classes and Objects.php
<?php
// PHP OOP - Classes e Objetos

// Definindo uma classe
class Pessoa {
    // Propriedades
    public $nome;
    public $idade;

    // Método construtor
    public function __construct($nome, $idade) {
        $this->nome = $nome;
        $this->idade = $idade;
    }

    // Método
    public function apresentar() {
        echo "Olá, meu nome é {$this->nome} e tenho {$this->idade} anos.<br>";
    }
}

// Criando objetos
$p1 = new Pessoa("Ana", 28);
$p2 = new Pessoa("Carlos", 35);

// Usando métodos do objeto
$p1->apresentar();
$p2->apresentar();
?>